pmd-java/src/test/java/net/sourceforge/pmd/lang/java/oom/DataStructureTest.java

```diff
@@ -30,13 +46,39 @@
  */
 public class DataStructureTest extends ParserTst {
 
+    MetricKey<ClassMetric> classMetricKey = new MetricKey<ClassMetric>() {
+        @Override
+        public String name() {
+            return null;
+        }
+
+
+        @Override
+        public ClassMetric getCalculator() {
+            return new RandomMetric();
+        }
+    };
+    MetricKey<OperationMetric> opMetricKey = new MetricKey<OperationMetric>() {
+        @Override
+        public String name() {
+            return null;
+        }
+
+
+        @Override
+        public OperationMetric getCalculator() {
+            return new RandomMetric();
+        }
+    };
     private PackageStats pack;
 
+
     @Before
     public void setUp() {
         pack = new PackageStats();
     }
 
+
```

Comment:
I've personally never see a 2-blank line style. :)

Stop reading now if you have more important things to do...

Back in the day I used to prefer `{` on a separate line, but I've since changed and come to value information density more.  I even tilt my monitors to portrait mode so I can see more LOC at once.  Not that I want to write huge classes or methods, but it does seem to greatly help my ability to _grok_ code faster, scrolling/navigation is my arch-nemesis. :)  I take mildly perverse pleasure whenever I write a complete meaningful class, JavaDoc and all, and it entirely fits within a single visual window on my monitor.  The definition is clearly arbitrary, but let's call this OSM, the one-screen-metric! ;)